### ISSUE TITLE
HTML output needs to comply with W3C validator

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-csd#81]
Using the W3C validator (https://validator.w3.org/) , the Metanorma HTML output (https://standards.calconnect.org/csd/cc-10005.html) is shown to have 50 errors and 32 warnings.

Somehow the document is given as `xhtml`, not `html`. The HTML should certainly be HTML5 compliant.

<img width="754" alt="Screen Shot 2019-05-29 at 4 24 20 PM" src="https://user-images.githubusercontent.com/11865/58541500-37ee9580-822e-11e9-9d9f-1612d34b665d.png">

Validating as HTML5:

<img width="759" alt="Screen Shot 2019-05-29 at 4 24 39 PM" src="https://user-images.githubusercontent.com/11865/58541517-4341c100-822e-11e9-85c6-7e5f229aacf1.png">

1. This is a Metanorma-wide problem. How do we fix this throughout?

2. I know @pierrefortier is good at hand-crafting HTML -- @opoudjis is there a way to let @pierrefortier fix up these issues?

**Comment by opoudjis** [comment]
2. Given how much of the HTML is generated by Nokogiri mapping, I don't see how.

1. I have no idea why generic HTML is being parsed as XHTML; and of course I've never had any reason to put in XHTML paraphernalia like DOCTYPE.

This is a requirement newly sprung on me after two years, and this is going to low priority.

**Comment by strogonoff** [comment]
@ronaldtse 

Can we have here some reference documents to be used for testing after markup is changed to ensure deliverables stay compliant in terms of layout & behavior?

It’s not inconceivable that as markup becomes valid it may be interpreted differently by software (e.g., a while back different doctypes were known to trigger different browser behaviors). Before we start, we should probably have:

* sample Metanorma projects (as source & rendered) to aid in testing that HTML / MS Word HTML / PDF deliverables look the same in the end, accompanied by
* rough guidelines as to which parts of the samples (if any) must stay pixel- (or mm-) perfect after markup changes.

Perhaps these things already exist, since they could be useful in ongoing Metanorma maintenance

**Comment by ronaldtse** [comment]
I'm going to transfer this to https://github.com/metanorma/metanorma because it is a project-wide task.
